buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1515,14 +1515,14 @@ class BeamModulePlugin implements Plugin<Project> {
       project.protobuf {
         protoc {
           // The artifact spec for the Protobuf Compiler
-          artifact = "com.google.protobuf:protoc:3.11.0" }
+          artifact = "com.google.protobuf:protoc:$protobuf_version" }
```

Comment:
applyPortabilityNature uses the vendored grpc artifacts and should use protoc/protoc-gen-grpc-java related to the vendored grpc version.